src/ucp/rma/rma_send.c

```diff
@@ -115,7 +115,7 @@ static void ucp_rma_request_bcopy_completion(uct_completion_t *self)
     ucp_request_t *req = ucs_container_of(self, ucp_request_t,
                                           send.state.uct_comp);
 
-    if (ucs_likely(req->send.length == req->send.state.dt.offset)) {
+    if (ucp_request_is_send_completed(req)) {
```

Comment:
where would we complete this request if comp.status != OK?

Reply:
on 119 line, `ucp_request_is_send_completed()` returns `1` if:
- either status is `UCS_OK`
- or `offset` == `length`